tests/haddock/libC.nix

```diff
@@ -17,7 +17,7 @@ let
   cabal = pkgs.writeText "libc.cabal" ''
     name: libc
     version: 0.1.0.0
-    build-type: simple
+    build-type: Simple
```

Comment:
Huh?

Reply:
My mistake I think.
Now, newer cabal seems to think it matters?

Reply:
I think it doesn't care, this just looks like noise in the diff :)

Reply:
Ah OK, then it's fine.